src/components/OptionsList/index.native.js

```diff
@@ -0,0 +1,17 @@
+import React, {forwardRef} from 'react';
+import {Keyboard} from 'react-native';
+import BaseOptionsList from './BaseOptionsList';
+
+export default forwardRef((props, ref) => (
+    <BaseOptionsList
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        ref={ref}
+
+        // Both `keyboardDismissMode` & `onScrollBeginDrag` props are needed to ensure that virtual keyboard is
+        // dismissed on all platforms.
+        // eslint-disable-next-line react/jsx-props-no-multi-spaces
+        keyboardDismissMode="on-drag"
+        onScrollBeginDrag={() => Keyboard.dismiss()}
```

Comment:
OK. This is fine. But as we are now abstracting away these props for native and web. Then it means we can get rid of the one which is not needed for native. You will get more info in the original PR that added this. I don't remember atm.

Reply:
Got it! We can get rid of `keyboardDismissMode="on-drag"`

[Reference commit](https://github.com/Expensify/App/commit/c72eb4add194cdee88261ab290da6de38cc98a51#diff-6a3f4819485fb409e83240f58a2b8ee3c096ad40718d8f9e12929c1adb888a38R238)